vision.md

```diff
@@ -1,4 +1,12 @@
-# Values
+# Our Vision
+
+## Mission and Vision
+
+*Hypha Worker Co-operative* cultivates collective growth and meaningful livelihoods through learning and building technologies together.
```

Comment:
_Hypha Worker Co-operative_ is italicized here. Is this why @patcon suggested the same in About Us with our legal name?

Reply:
heh not specifically. I like doing it, but recognize it's not a convention. I'm fine to leave it to dc, and I'll stop italicising in handbook unless I hear otherwise

Reply:
```suggestion
Hypha Worker Co-operative cultivates collective growth and meaningful livelihoods through learning and building technologies together.
```